src/fileutils.cpp

```diff
@@ -57,3 +57,19 @@ std::vector<std::string> listDirectory(std::string sDirectory) {
    sort(s.begin(), s.end()); // sort elements
    return s;
 }
+
+// Returns a vector containing the names of the files having extension "ext" in
+// the specified directory
+std::vector<std::string> listDirectoryExt(std::string sDirectory, std::string ext) {
```

Comment:
Pass sDirectory and ext as `const std::string&`.
We could rename sDirectory just directory here and in listDirectory.